addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -62,12 +64,43 @@
 
     private List<DeviceMessageListener> deviceStatusListeners = new CopyOnWriteArrayList<>();
 
-    private static byte seqNbr = 0;
-    private static RFXComTransmitterMessage responseMessage = null;
-    private Object notifierObject = new Object();
     private RFXComBridgeConfiguration configuration = null;
     private ScheduledFuture<?> connectorTask;
 
+
+    private class TransmitQueue {
+        private LinkedBlockingQueue<RFXComBaseMessage> queue = new LinkedBlockingQueue<RFXComBaseMessage>();
```

Comment:
A quick check suggest that you could use the type as an `Queue` its a common best-practice (in Java) to use an interface for fields which is as common as possible.